package.json

```diff
@@ -14,6 +14,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-hook-form": "^7.38.0",
```

Comment:
Usually in a real-world project it's best to lock the packages versions by removing `^`. Here it is not a big deal, just a note.

Reply:
Removed them